### DISCUSSION TITLE
Pentiar Suntouch support?

### DISCUSSION THREAD

**Discussion by BlackAlgae** [tagyoureit/nodejs-poolController#997] [General]
Does it support the suntouch panel? Thanks in advance

**Comment by tagyoureit** [comment]
It's limited, and you might need to flash the firmware.  https://github.com/tagyoureit/nodejs-poolController/discussions/777

**Reply by BlackAlgae** [reply]
Hi thanks for the reply! Really appreciate the insight.

By limited what am I able to do and not do? Im only looking for the essentials like spa/pool mode, turn on/off lights, and heater. 

From what I read it looks like the gentlemen disconnected the comms from the suntouch and the pool? if my firmware matches his at 2.06 will i have the same luck? 

Many thanks!
